druid/src/window.rs

```diff
@@ -215,6 +226,17 @@ impl<T: Data> Window<T> {
 
         self.post_event_processing(queue, data, env, false);
 
+        //In some platforms, timer tokens are reused. So it is necessary to remove token from
+        //window's timer before adding base state's timers to it.
+        if let Event::Internal(InternalEvent::RouteTimer(token, _)) = event {
+            self.timers.remove(&token);
+        }
+
+        //If at least one widget requested timer, collect those timers from widgets and add to window's timers map.
```

Comment:
*If at least one widget requested **a** timer, ~collect those timers from widgets and~ add **all the requested timers** to window's timers map.*